yaml/prometheus-svc.yml

```diff
@@ -1,16 +1,17 @@
+---
+---
 apiVersion: v1
 kind: Service
 metadata:
-  name: prometheus
-  namespace: openfaas
   labels:
-    app: prometheus
+    app: openfaas
+    component: prometheus
+  name: prometheus
+  namespace: "openfaas"
 spec:
-  type: NodePort
+  type: ClusterIP
   ports:
     - port: 9090
       protocol: TCP
-      targetPort: 9090
```

Comment:
Please update any docs which mention a NodePort of 31119 and give the kubectl port-forward equivalent before the merge of this PR.

Reply:
I could not find any explicit reference to this port

Reply:
actually just found one in the docs, my search missed that repo.  The only reference is here, and it doesn't really say how to use it.  I could just remove the line?

Reply:
I added a small section to the chart readme to describe accessing prometheus, we will need a separate PR to the docs repo to replicate or remove the instructions there too

Reply:
It's also in the readme potentially.

Reply:
I checked the readme of faas-netes and the chart and then in the docs project, it was only directly referenced in the docs project